scripts/run.py

```diff
@@ -363,7 +363,7 @@ def run_classpath_tool(project_name, project_path):
     P = Pipeline()
     P.info("[Analysis][%s] Generating classpath..." % project_name)
     title = "Classpath generation"
-    command = "sbt -batch gcp" 
+    command = """sbt -batch "show test:fullClasspath" | sed -n -E 's/Attributed\(([^)]*)\)[,]?/\\n\\1\\n/gp' | grep "/home" > classpath.dat"""
```

Comment:
What do you think - would it be more portable to have it `grep "^/"` (will work on OSX and Linux) :-)